hack/chart-verifier-docker.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# KUBECONFIG is either the value the user has configured in the
+# environment or the default location `kubectl` uses.
+KUBECONFIG="${KUBECONFIG:-${HOME}/.kube/config}"
+
+# CHART_VERIFIER_IMAGE is the location of the image to use to verify
+# the chart.
+CHART_VERIFIER_IMAGE="${CHART_VERIFIER_IMAGE:-quay.io/redhat-certification/chart-verifier:main}"
+
+# DOCKER contains the path for the `podman` program.
+DOCKER="${DOCKER:-/usr/bin/docker}"
```

Comment:
let's try to detect if `docker` is available on PATH and use it, instead of expecting on a certain location, for example:

```
for cr in podman docker; do
  [ -z "$CONTAINER_RUNTIME" ] && $cr --help >/dev/null 2>&1 && CONTAINER_RUNTIME=$cr
done;

```